etc/otel-jmx.config.yaml

```diff
@@ -12,25 +11,46 @@ rules:
        metric: kafka.streams.records.lag
        type: gauge
        desc: the current lag of the partition
-       unit: '{messages}'
+       unit: '{records}'
  - bean: kafka.consumer:type=consumer-fetch-manager-metrics,client-id=*
    metricAttribute:
      clientid: param(client-id)
    mapping:
      records-lag-max:
        metric: kafka.streams.records.lag.max
        type: gauge
-       desc: the current lag of the partition
-       unit: '{messages}'
- - bean: kafka.streams:type=stream-thread-metrics,thread-id=*
+       desc: the max lag of all partitions
+       unit: '{records}'
+ - bean: kafka.consumer:type=consumer-coordinator-metrics,client-id=*
```

Comment:
sorry, git diff is a bit messed up -- I didn't remove or change any metrics (other than the minor fixes right above this), just organized them according to the client/level